From 339bea257c5f7b8e1c9afffc277ebf7047cfde72 Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Fri, 14 Jul 2023 06:36:47 +0300
Subject: [PATCH] [FSW-402] fs1: compile arcboot for fs1

This patch adds arcboot as part of fs1_arc
target config.

Change-Id: Ibab6f24f0f62125b2699471d449548c198d80b77
---

diff --git a/configs/fs1_arc_defconfig b/configs/fs1_arc_defconfig
index b9ab823..6088722 100644
--- a/configs/fs1_arc_defconfig
+++ b/configs/fs1_arc_defconfig
@@ -25,3 +25,4 @@
 BR2_PACKAGE_HOST_UBOOT_TOOLS=y
 BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SUPPORT=y
 BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SIGNATURE_SUPPORT=y
+BR2_PACKAGE_ARCBOOT=y
